.travis.yml

```diff
@@ -13,7 +14,6 @@ matrix:
       dist: xenial
     - python: 3.6
       env: TOXENV=flake8
-    - python: 3.6
```

Comment:
Note: This is covered by line 5. When "python:" only has one element it did not seem to get picked up by Travis so the workaround was to explicitly add it to the matrix. Now with python 2.7 also added to the list, keeping it in the matrix as well causes duplication, so removing it for now.